packages/layers/base/nuxt.config.ts

```diff
@@ -19,7 +19,7 @@ export default defineNuxtConfig({
   compatibilityDate: '2025-07-10',
 
   imports: {
-    dirs: ['interfaces']
+    dirs: ['interfaces', 'composables']
```

Comment:
Composables should be extended automatically right? Do we need to specify it here?